assets/schema/ensemble_schema.json

```diff
@@ -2190,6 +2190,10 @@
                   "type": "string",
                   "description": "Wallet connect project Id, get it from wallet connect dashboard"
                 },
+                "wallet": {
+                  "type": "string",
+                  "description": "Optional for Android, Required in IOS, when specified walletconnect will open and connect to specified wallet. e.g metamask, trust, rainbow..."
```

Comment:
perhaps we should make it required for both iOS and Android otherwise people will test their app on Android and it works and doesn't work on iOS. What do you think?

Reply:
Can do, for android we can add new type `any` to open android tray of all wallet connection and for IOS any won't work and we'll show error for specific wallet